Classes/Service/Eid.php

```diff
@@ -24,72 +24,10 @@
  *  This copyright notice MUST APPEAR in all copies of the script!
  ***************************************************************/
 
-use TYPO3\CMS\Core\Utility\GeneralUtility;
+\TYPO3\CMS\Frontend\Utility\EidUtility::initTCA();
 
-/**
```

Comment:
Looks like the class-definition have been remove..

Reply:
The class has now been moved to its own file. The eID script and processor class now reside in individual files.